### DISCUSSION TITLE
資料爭奪戰：AI 產業所需的資料有多重要？

### DISCUSSION THREAD

**Discussion by Tina-Peng-1566** [1101-GS4719-C/Python-Community-GS4719-C#2053] [General]
https://infosecu.technews.tw/2023/11/13/how-important-is-the-data-required-by-the-ai-%E2%80%8B%E2%80%8Bindustry/   

先前關注的是資料版權問題，不管是新聞、Yuotube影片等各種文與作品，只要未經授權就被拿去訓練AI的，都會有此爭議，可見擁有大量且高品質的資料在未來是很重要的，因為「資料成為了衡量一個國家或企業競爭力的新標準，未來的商業模式和科技創新無疑將圍繞著資料展開」。

我認為Adobe就轉型得很好，在AI生圖軟體的快速進步下，運用原有的龐大資料庫發展出自家的AI系統，不但跟上時代的潮流，也保住其在創意軟體領域的龍頭位置，對於一個相對於AI來說較「傳統」的產業，能同時做到這兩件事並不簡單。

**Comment by hunteryang0108** [comment]
我認為需要龐大資料的全知型AI需求正在慢慢降低，現在各大企業更關注小型語言模型的發展。如何從更少的資訊中回答問題成為各個語言模型需要挑戰的難題，未來這類AI將會被分別應用在特定的專屬領域，而不用回答使用者的所有問題。

**Comment by Jerryguo888** [comment]
資料版權爭議像我們展示了未來高品質數據的重要性，商業模式和科技創新將圍繞數據展開，而Adobe的轉型，運用龐大資料庫發展AI系統，使其跟上AI時代潮流，也展示了傳統產業在數位化中的突破。
